src/plugins/dashboard/public/services/application/application_service.ts

```diff
@@ -24,6 +24,7 @@ export const applicationServiceFactory: ApplicationServiceFactory = ({ coreStart
       getUrlForApp,
       capabilities: { advancedSettings, maps, navLinks, visualize },
     },
+    customBranding,
```

Comment:
Why does application service return customBranding when its also returned custom_branding service? Is returning customBranding needed in ApplicationService?

Reply:
another good catch, I think this is leftover from a previous PR - removed 👍🏻